Ginger/GingerTest/POMs/JavaPOM.cs

```diff
@@ -0,0 +1,404 @@
+            mGR.RunAction(actUIElement);
+
+            //Assert
+            Assert.AreEqual(actUIElement.Status, Amdocs.Ginger.CoreNET.Execution.eRunStatus.Passed);
+        }
+
+        private static void StartAgent()
+        {
+            if (mGR == null)
+            {
+
+                mGR = new GingerRunner();
+                mGR.CurrentSolution = new Ginger.SolutionGeneral.Solution();
+                mBF = new BusinessFlow();
+                mBF.Activities = new ObservableList<Activity>();
+                mBF.Name = "BF Test Java Driver";
+                Platform p = new Platform();
+                p.PlatformType = ePlatformType.Java;
+                mBF.TargetApplications.Add(new TargetApplication() { AppName = "JavaTestApp" });
+                Activity activity = new Activity();
+                activity.TargetApplication = "JavaTestApp";
+                mBF.Activities.Add(activity);
+                mBF.CurrentActivity = activity;
+
+                ActLaunchJavaWSApplication LJA = new ActLaunchJavaWSApplication();
+                LJA.LaunchJavaApplication = true;
+                LJA.LaunchWithAgent = true;
+                LJA.WaitForWindowTitle = "Java Swing";
+                LJA.AddOrUpdateInputParamValue(ActLaunchJavaWSApplication.Fields.PortConfigParam, ActLaunchJavaWSApplication.ePortConfigType.Manual.ToString());
+                LJA.Port = "9898";
+                LJA.URL = TestResources.GetTestResourcesFile(@"JavaTestApp\JavaTestApp.jar");
+                activity.Acts.Add(LJA);
+                mGR.PrepActionValueExpression(LJA);
+                LJA.Execute();
+                // TODO: add wait till action done and check status
+                //if (!string.IsNullOrEmpty(LJA.Error))
+                //{
+                //   throw new Exception(LJA.Error);
+                //}
+
+                mDriver = new JavaDriver(mBF);
+                mDriver.JavaAgentHost = "127.0.0.1";
+                mDriver.JavaAgentPort = 9898;
+                mDriver.CommandTimeout = 120;
+                mDriver.cancelAgentLoading = false;
+                mDriver.DriverLoadWaitingTime = 30;
+                mDriver.ImplicitWait = 30;
+                
+                //Agent a = new Agent();
+                //a.Active = true;
+                //a.DriverType = Agent.eDriverType.JavaDriver;
+
+                //a.Name = "Java Agent";
+                //a.Driver = mDriver;
+                mJavaAgent.Driver = mDriver;
+                mDriver.StartDriver();
+                mGR.SolutionAgents = new ObservableList<Agent>();
+                mGR.SolutionAgents.Add(mJavaAgent);
+
+                ApplicationAgent AA = new ApplicationAgent();
+                AA.AppName = "JavaTestApp";
+                AA.Agent = mJavaAgent;
+                mBF.CurrentActivity.CurrentAgent = mJavaAgent;
+                mGR.ApplicationAgents.Add(AA);
+                mGR.CurrentBusinessFlow = mBF;
+                //mJavaAgent.StartDriver();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4423505)